src/views/credits/credits.jsx

```diff
@@ -196,16 +191,6 @@ var Credits = React.createClass({
                 <p><FormattedMessage id='credits.acknowledgementsCommunity' /></p>
                 <p><FormattedMessage id='credits.acknowledgementsInfluencers' /></p>
 
-                <h2><FormattedMessage id='credits.supportersTitle' /></h2>
```

Comment:
I don't think you intended to remove these lines :) We definitely want to keep recognizing organizations that support Scratch.